SUPPORT.md

```diff
@@ -0,0 +1,30 @@
+# Support
+
+Thank you for your interest in this project! If you are experiencing problems
+or have questions, the following are some suggestions for how to get help.
+
+> [!NOTE]
+> Before participating in our community, please read our [code of
+> conduct](CODE_OF_CONDUCT.md). By interacting with this repository,
+> organization, or community, you agree to abide by its terms.
+
+## Report an issue or request a feature
+
+To report an issue or request a feature in Qualtran, please first search the
+[issue tracker on GitHub](https://github.com/quantumlib/Qualtran/issues) to
+check if there is already an open issue identical or similar to your bug
+report/feature request. If there is none, go ahead and file a new issue in the
+issue tracker.
+
+## Attend the developer meetings
+
+_Qualtran Cynq_ is our biweekly virtual meeting of contributors to discuss
```

Comment:
I think we usually spell sync normally for this project :)

Reply:
Oh, LOL!

And of course I had auto-merge turned on, which means I'll have to do a separate PR to fix this. Serves me right for working too fast …